### DISCUSSION TITLE
Disable spam protection

### DISCUSSION THREAD

**Discussion by crbast** [anonaddy/anonaddy#329] [Feature Requests]
A lot of services is marked as spam (for example : service@paypal.ch). 

It could be a great feature to enable/disable SPAM protection on some alias (not all, just some specific ones).

**Comment by willbrowningme** [comment]
When you say they are marked as spam where do you see this? Do you mean it appears on the failed deliveries page as being rejected for "spam"?

**Reply by crbast** [reply]
Hello, thank you for your response. I mean on the failed-deliveries page. But after investigation it seems to be an issue with the destination email (with strict spam protection)

**Reply by willbrowningme** [reply]
Yes, that is nothing to do with AnonAddy rejecting your messages. It is your mail provider e.g. Google, Outlook, Apple or whoever you use.

In order to reduce the chance of a message being rejected your best option is to enable encryption to prevent them scanning it - https://anonaddy.com/faq/#im-not-receiving-any-emails-whats-wrong
